src/lib.rs

```diff
@@ -7,7 +7,103 @@
 //! * [Documentation](https://tailhook.github.io/gron/gron/index.html)
 
 extern crate rustc_serialize;
+extern crate serde_json;
 
-mod json_struct;
+pub mod for_rustc_serialize;
+pub mod for_serde;
+
+use std::io::{self, Write};
+
+//pub use for_rustc_serialize::json_to_gron as rustc_serialize_json_to_gron;
+//pub use for_serde::json_to_gron as serde_json_to_gron;
+
+
+pub trait ToGron {
+    fn to_gron<W: Write>(&self, out: &mut W, prefix: &str) -> io::Result<()>;
+}
+
+pub fn json_to_gron<W: Write, T: ToGron>(out: &mut W, prefix: &str, json: &T)
```

Comment:
This function lost it's docstring. This is a most important docstring in the module!